web/src/app/wizard/WizardScheduleForm.js

```diff
@@ -198,6 +199,18 @@ export default class WizardScheduleForm extends React.Component {
                     </InputAdornment>
                   ),
                 }}
+                // keyboard picker props
+                format='MM/dd/yyyy, hh:mm a'
```

Comment:
Since we're setting this everywhere, and all the others, should we just be wrapping the picker instead?

Seems like it would be easy for all these pickers to get out-of-sync (e.g. if we change the mask)

Also, is there a way we can make this international friendly? IIRC the US is the only place that does MDY

Reply:
We could probably put effort into making it dynamically change the format depending on where they live, but with the placeholder and defaults stored in the field I don't think it's necessary/worth the time here to try and do something like this. Could be a nice improvement later on down the road though.

What do you mean by "wrapping the picker" with a format string, though? I'm not sure I fully understand. I can use an exported const for the date-time format, similar to our poll intervals.

Reply:
Currently we are displaying dates as something like `December 12` for pickers, which side-steps the issue since the month is spelled out.

Changing the display format to one that isn't correct for the current locale will cause confusion -- `01/02/2020` does not mean the same thing in all countries, so hard-coding it is problematic (Feb 1st or Jan 2nd) because we need to support multiple time-zones/countries.

Wrapping it is because we're passing the same mask, placeholder, validation, etc.. in a lot of places -- if the defaults of the keyboard picker aren't usable, we should wrap it so we don't need to copy/paste and keep-in-sync all those options.